src/components/FormInput.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const FormInput = ({
+  idAttribute, inputValue, inputPlaceHolder, onChange, inputType, classAttribute
```

Comment:
Attribute names should be standard.
Update to `id, value, placeHolder, onChange, type, class`

Reply:
I have updated the attribute names, but I could not update the **classAttribute** to **class** because **class** is a reserved keyword in javascript so I updated it to **className** instead.